src/dto/notes/update-note.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { Transform, TransformFnParams } from 'class-transformer';
+import { Length, IsOptional, IsUUID, IsDate, MinDate } from 'class-validator';
+
+import {
+  MIN_DATE_VALUE,
+  TASK_MAX_LENGTH,
+  TASK_MIN_LENGTH,
+  UUID_LENGTH,
+} from 'common';
+
+export class UpdateNoteDto {
+  @Length(UUID_LENGTH)
+  @IsUUID()
+  id: string;
+
+  @Length(TASK_MIN_LENGTH)
+  title: string;
+
+  @Length(TASK_MIN_LENGTH, TASK_MAX_LENGTH)
+  content: string;
+
+  @IsOptional()
+  @IsDate()
+  @Transform(({ value }: TransformFnParams) => new Date(value))
```

Comment:
should be in separate util

Reply:
i remove this field  because it is unnecessary